spec.md

```diff
@@ -0,0 +1,228 @@
+
+### Layer Versioning
+
+Layer formats that may change should define a version to future proof new enhancements. How the artifact tooling processes forward and backward compatibility is up to the artifact and tooling authors.
+
+### Defining `layer.mediaTypes`
+
+`layer.mediaTypes` fall into 2 categories:
+
+1. **Existing generic formats**: as defined by [IANA][iana-media-types].  
+    These types are neither artifact nor `vendor|org|entity` specific, nor are they versioned.  
+    Generic formats enable simple single layer scenarios, or where there's little to no differentiation between the different layers.
+    - [text/*](https://www.iana.org/assignments/media-types/media-types.xhtml#text)
+    - [application/json](https://www.iana.org/assignments/media-types/application/json)
+    - [application/xml](https://www.iana.org/assignments/media-types/application/xml)
+1. **Vendor|Org|Entity definitions**: identify different versioned layer types within the same artifact.  
+    Unique definitions enable layers to be handled individually without assuming ordered positioning. Each layer may use an existing [IANA][iana-media-types] format, but represent different usages within the artifact type.  
+    A Registry Doc artifact (`application/vnd.oci.sample-repo-doc.config.v1`) which represents the contents of a repo cloud be represented as:
+    - `application/vnd.oci.sample-repo-doc.summary.layer.v1.md` - used as the readme of a repo, displayed on a registry webpage.
+    - `application/vnd.oci.sample-repo-doc.content.layer.v1.tar` - provides detailed documentation, including multiple `.md` files for offline reading.
+
+**Vendor|Org|Entity definitions** utilize the `layer.mediaType` with the following format:  
+`[registration-tree]`.`[org|company|entity]`.`[layerType]`.`[optional-layerSubType]`.layer.`[version]`.`[fileFormat]`+`[optional-compressionFormat]`
+
+- **`registration-tree`** - represents an IANA root tree. See [iana registration trees][iana-trees] for public and private tree options.
+- **`org|company|entity`** - represents an open source foundation (`oci`, `cncf`), a company or some unique entity.
+- **`layerType`** - a value representing the short name of the type.
+- **`optional-layerSubType`** - provides additional extensibility of a `layerType`
+- **`version`** - provides artifact authors the ability to revision their schemas and formatting, enabling tools to identify which format they will process. The content format (json, yaml) may not revision, but the content within the layer may evolve. Using a version enables artifact tools to identify the format for processing.
```

Comment:
I don't follow the part about format "may not revision" in the context of versioning of the content. do you mean format type can't change from json to yaml?

Reply:
I've removed the confusing text.